lib/statsample-glm/glm/formula.rb

```diff
@@ -29,9 +29,11 @@ def parse_formula(form = :token)
 
       private
 
-      def add_contant_term_if_required
-        # TODO: Add support for constants
-        @tokens.unshift Token.new('1')
+      def manage_contant_term
+        @tokens.unshift Token.new('1') unless
+          @tokens.include?(Token.new('1')) ||
+          @tokens.include?(Token.new('0'))
+        @tokens.delete Token.new('0')
```

Comment:
Can we use this way to sort the tokens with `Token.new '1'` involved? Currently I do `@tokens.sort` to sort the tokens so that `Token.new '`'` appears in the front. I don't think it will be possible to do the same using `Token` method or would it?